src/config-schema.coffee

```diff
@@ -21,7 +21,6 @@ module.exports =
       followSymlinks:
         type: 'boolean'
         default: true
-        title: 'Follow symlinks'
```

Comment:
It was, te default title is the key (`followSymlinks`) capitalized, which would be `Follow Symlinks`. The downcase `s` was inconsistent with the other titles.